### ISSUE TITLE
Determine system for storing confidential employee information

### ISSUE THREAD

**Issue by patcon** [hyphacoop/organizing#109]
Re-ticketed from https://github.com/hyphacoop/organizing/issues/99#issuecomment-530122928

## Shared Description

:speaking_head: **Loomio:** N/A  
:date: **Due date:** N/A  
:dart: **Success criteria:** We have a place to store data, which everyone is aware of and can live with.  

[Chat context](https://matrix.to/#/!wvFLvXHaNrxWWwFOKj:tomesh.net/$15682267971724QblbX:tomesh.net?via=tomesh.net&via=matrix.org&via=elon.li)

We eventually need a place to store SIN's for employees, and other comparably sensitive information.

[CRA: SIN Code of Practice Section 2 - SIN holders' responsibilities](https://www.canada.ca/en/employment-social-development/services/sin/reports/code-of-practice/section-2.html)

> You must provide your SIN [...] to your employer **within three (3) days** after the day on which your employment begins. [...] In the private sector, you must provide your SIN **to your employer for income tax and benefit purposes.**

To be fair, we can avoid it for now, and are NOT blocked on #99, just can't authorize other directors on account, and each need to call in manually (~30 min wait time for each).

## To Do

- [x] Browse the full [SIN Code of Practice](https://www.canada.ca/en/employment-social-development/services/sin/reports/code-of-practice.html)?
  - [x] Closely read [Sec 3: Employers' responsibilities](https://www.canada.ca/en/employment-social-development/services/sin/reports/code-of-practice/section-3.html)?
  - [x] ~~summarize?~~ linked to in handbook
- [x] Discuss technical concerns at @hyphacoop/infra-ops-wg meeting
- [x] Seek advice from accountant (added to [kickoff agenda](https://hackmd.io/@patcon/ryMxS3okB/https%3A%2F%2Fhackmd.io%2F716xTfHcQ-CSzwmcYgmtjw%3Fview?type=book))
- [x] Discuss amongst @hyphacoop/finance-wg members

**Comment by patcon** [comment]
I stubbed out some possible todos, but open to feedback on whether those seem right :)

Systems we might use to store SIN, if we choose:
- Google Spreadsheet
- https://pass.hypha.coop
- ...

As per chat, we could choose to only authorize a sub-set of member-owners to access this information, to limit risk. (Finance WG members? to-be-named Treasurer? or Directors? other???)

**Comment by dcwalk** [comment]
From discussion in chat, it appears we want to widen the net of systems beyond only digital storage of SINs.

**Comment by dcwalk** [comment]
I flagged also that we've been through a process of research on this before as part of the Business Planning WG, here is the consolidated notes on resources from our business plan:

Sources: Your guide to the Employment Standards Act  https://www.ontario.ca/document/your-guide-employment-standards-act-0
   - Records Keeping https://www.ontario.ca/document/your-guide-employment-standards-act-0/record-keeping
Ontario Ministry of Labour - What Businesses Need to Know
https://www.labour.gov.on.ca/english/es/pubs/fs_compliance.php
Office of the Privacy Commissioner - Privacy in the Workplace  https://www.priv.gc.ca/en/privacy-topics/employers-and-employees/
IPC (Ontario) - Your Privacy Rights https://www.ipc.on.ca/privacy/your-privacy-rights/

**Comment by benhylau** [comment]
@patcon @dcwalk it seems we have already decided on a process. Is there a reason why we cannot close this out? It's unlikely we'll revisit this unless something comes up.

**Comment by dcwalk** [comment]
Well, we've decided for SINs, but I think we're not clear on our actual responsibilities as employers, in particular for financial and employee data, do you all in finance WG have that locked down?

May want to reframe this issue

**Comment by patcon** [comment]
> We eventually need a place to store SIN's for employees, _and other comparably sensitive information._

I have a habit of over-scoping things. This was created mainly to address SIN, which is the main thing people had opinions on that we needed to find solution for. I could see other conversations being a new ticket to open when it feels urgent, but might not expect that here. (ie. no objection to closing, nor reframing)

Admittedly, not sure where responsibility for record-keeping know-how best resides, and who advocates for it -- finance or operations.

**Comment by dcwalk** [comment]
Happy to close this out, but I'd favour ensuring finance (which has their own obligations) and operations (which has slightly less ones) has a handle on record-keeping.... so maybe a new issue is required :grin:

**Comment by benhylau** [comment]
I have done `Closely read Sec 3: Employers' responsibilities?` and checked off all the things that I believe have already happened, and personally see no need to spin off another issue until something comes up. Assigned this to @patcon to close or spin new issue if he sees fit.

**Comment by patcon** [comment]
> I'd favour ensuring finance (which has their own obligations) and operations (which has slightly less ones) has a handle on record-keeping

Trying to think how to close this while capturing the above somewhere concrete, which does feel important.

@dcwalk What if we added "familiarity with record-keeping docs x, y , z" to appropriate role descriptions? As in, if we added some outline of what docs the stewards of Finance or Operations should be comfortable with. So a person entering that role would know if they're prepped -- what docs to read, videos to watch, etc.

For example, Finance role is a thing that's being developed (see linked issue above).

Would this sort of capture make this closable?
